src/katas/katas.h

```diff
@@ -1,17 +1,34 @@
 #ifndef CLINGS_KATAS_H
 #define CLINGS_KATAS_H
 
+#include "types/types.h"
+#include "stdbool.h"
 
-// TODO: extract this to yaml
-static const char * kata_folder_path = "../katas/";
+typedef struct Kata Kata;
+struct Kata {
```

Comment:
We need a convention for types

either we do
types_like_this_t
or 
TypesLikeThis

I'm okay with the one you did, but then we should use camelCase for variable names shouldn't we ?

Reply:
Finally I think we should use camelCase